skrf/calibration/deembedding.py

```diff
@@ -0,0 +1,80 @@
+from ..frequency import *
+from ..network import *
+
+class Deembedding(object):

```

Comment:
I did try subclassing `calibration` but it has `ideals` as a compulsory input. De-embedding only requires a bunch of `network` objects that will be removed from the raw device measurement. So I decided to not subclass `calibration` and went with a separate class for de-embedding. What are your thoughts on this?

Great point on using Abstract classes. I will use that for the base class because it will never be instantiated directly.